nwb_conversion_tools/utils/json_schema.py

```diff
@@ -12,7 +12,7 @@
 
 def get_base_schema(tag=None, root=False, id_=None, **kwargs) -> dict:
     """Return the base schema used for all other schemas."""
-    base_schema = dict(required=[], properties={}, type="object", additionalProperties=False)
+    base_schema = dict(required=[], properties={}, type="object", additionalProperties=True)
```

Comment:
Ahh, I see, you had to flip this flag to allow the `ndx_dandi_icephys` extension metadata?

I'd recommend instead restoring this outer property key to False, nesting the `ndx_dandi_icephys` under an `Icephys` level, and allowing the extra properties for that level specifically.

Otherwise goodness knows what very general mistakes might pop up from improperly nested metadata structures.